### ISSUE TITLE
integration tests test_ttl_replicated/test.py::test_ttl_compatibility assert_eq_with_retry retry 1 exception Client failed! Return code: 210, stderr: Code: 210. DB::NetException: Connection refused (172.18.0.6:9000)

### ISSUE THREAD

**Issue by songenjie** [ClickHouse/ClickHouse#22066]
```py
test_multiple_disks/test.py 
        Can't find a suitable configuration file in this directory or any
        parent. Are you in the right directory?

        Supported filenames: docker-compose.yml, docker-compose.yaml
        

        Can't find a suitable configuration file in this directory or any
        parent. Are you in the right directory?

        Supported filenames: docker-compose.yml, docker-compose.yaml
        
Cluster start called. is_up=False, destroy_dirs=True
Trying to kill unstopped containers...
Unstopped containers killed
Setup directory for instance: node1 destroy_dirs: True
Copy common default production configuration from /clickhouse-config
Create directory for configuration generated in this helper
Create directory for common tests configuration
Copy common configuration from helpers
Generate and write macros file
Copy custom test config files ['/ClickHouse/tests/integration/test_multiple_disks/configs/logs_config.xml', '/ClickHouse/tests/integration/test_multiple_disks/configs/config.d/storage_configuration.xml', '/ClickHouse/tests/integration/test_multiple_disks/configs/config.d/cluster.xml'] to /ClickHouse/tests/integration/test_multiple_disks/_instances/node1/configs/config.d
Setup database dir /ClickHouse/tests/integration/test_multiple_disks/_instances/node1/database
Setup logs dir /ClickHouse/tests/integration/test_multiple_disks/_instances/node1/logs
Env {'LLVM_PROFILE_FILE': '/var/lib/clickhouse/server_%h_%p_%m.profraw'} stored in /ClickHouse/tests/integration/test_multiple_disks/_instances/node1/env_file
Entrypoint cmd: bash -c "clickhouse server --config-file=/etc/clickhouse-server/config.xml --log-file=/var/log/clickhouse-server/clickhouse-server.log --errorlog-file=/var/log/clickhouse-server/clickhouse-server.err.log --daemon; tail -f /dev/null"
Setup directory for instance: node2 destroy_dirs: True
Copy common default production configuration from /clickhouse-config
Create directory for configuration generated in this helper
Create directory for common tests configuration
Copy common configuration from helpers
Generate and write macros file
Copy custom test config files ['/ClickHouse/tests/integration/test_multiple_disks/configs/logs_config.xml', '/ClickHouse/tests/integration/test_multiple_disks/configs/config.d/storage_configuration.xml', '/ClickHouse/tests/integration/test_multiple_disks/configs/config.d/cluster.xml'] to /ClickHouse/tests/integration/test_multiple_disks/_instances/node2/configs/config.d
Setup database dir /ClickHouse/tests/integration/test_multiple_disks/_instances/node2/database
Setup logs dir /ClickHouse/tests/integration/test_multiple_disks/_instances/node2/logs
Env {'LLVM_PROFILE_FILE': '/var/lib/clickhouse/server_%h_%p_%m.profraw'} stored in /ClickHouse/tests/integration/test_multiple_disks/_instances/node2/env_file
Entrypoint cmd: bash -c "clickhouse server --config-file=/etc/clickhouse-server/config.xml --log-file=/var/log/clickhouse-server/clickhouse-server.log --errorlog-file=/var/log/clickhouse-server/clickhouse-server.err.log --daemon; tail -f /dev/null"
Setup ZooKeeper
get_instance_ip instance_name=zoo1
get_instance_ip instance_name=zoo2
get_instance_ip instance_name=zoo3
All instances of ZooKeeper started
('Trying to create ClickHouse instance by command %s', 'docker-compose --project-name roottestmultipledisks --file /ClickHouse/tests/integration/test_multiple_disks/_instances/node1/docker-compose.yml --file /compose/docker_compose_zookeeper.yml --file /ClickHouse/tests/integration/test_multiple_disks/_instances/node2/docker-compose.yml up -d --no-recreate')
ClickHouse instance created
get_instance_ip instance_name=node1
Waiting for ClickHouse start...
ClickHouse started
get_instance_ip instance_name=node2
Waiting for ClickHouse start...
ClickHouse started
.............FF..Syncing replica
....assert_eq_with_retry retry 1 exception Client failed! Return code: 210, stderr: Code: 210. DB::NetException: Connection refused (172.18.0.6:9000)


assert_eq_with_retry retry 2 exception Client failed! Return code: 210, stderr: Code: 210. DB::NetException: Connection refused (172.18.0.6:9000)


.....Mutation failed
.Mutation failed
.assert_eq_with_retry retry 1 exception Client failed! Return code: 210, stderr: Code: 210. DB::NetException: Connection refused (172.18.0.6:9000)


assert_eq_with_retry retry 2 exception Client failed! Return code: 210, stderr: Code: 210. DB::NetException: Connection refused (172.18.0.6:9000)


.assert_eq_with_retry retry 1 exception Client failed! Return code: 210, stderr: Code: 210. DB::NetException: Connection refused (172.18.0.6:9000)


assert_eq_with_retry retry 2 exception Client failed! Return code: 210, stderr: Code: 210. DB::NetException: Connection refused (172.18.0.6:9000)


...........assert_eq_with_retry retry 1 exception Client failed! Return code: 210, stderr: Code: 210. DB::NetException: Connection refused (172.18.0.6:9000)


assert_eq_with_retry retry 2 exception Client failed! Return code: 210, stderr: Code: 210. DB::NetException: Connection refused (172.18.0.6:9000)


.assert_eq_with_retry retry 1 exception Client failed! Return code: 210, stderr: Code: 210. DB::NetException: Connection refused (172.18.0.6:9000)


assert_eq_with_retry retry 2 exception Client failed! Return code: 210, stderr: Code: 210. DB::NetException: Connection refused (172.18.0.6:9000)


.

=================================== FAILURES ===================================
__ test_concurrent_alter_move_and_drop[concurrently_dropping_mt-MergeTree()] ___

start_cluster = <helpers.cluster.ClickHouseCluster object at 0x7f625cbbdba8>
name = 'concurrently_dropping_mt', engine = 'MergeTree()'

    @pytest.mark.parametrize("name,engine", [
        ("concurrently_dropping_mt", "MergeTree()"),
        ("concurrently_dropping_replicated_mt",
         "ReplicatedMergeTree('/clickhouse/concurrently_dropping_replicated_mt', '1')",),
    ])
    def test_concurrent_alter_move_and_drop(start_cluster, name, engine):
        try:
            node1.query("""
                CREATE TABLE {name} (
                    EventDate Date,
                    number UInt64
                ) ENGINE = {engine}
                ORDER BY tuple()
                PARTITION BY toYYYYMM(EventDate)
                SETTINGS storage_policy='jbods_with_external'
            """.format(name=name, engine=engine))
    
            values = list({random.randint(1, 1000000) for _ in range(0, 1000)})
    
            def insert(num):
                for i in range(num):
                    day = random.randint(11, 30)
                    value = values.pop()
                    month = '0' + str(random.choice([3, 4]))
                    node1.query("INSERT INTO {} VALUES(toDate('2019-{m}-{d}'), {v})".format(name, m=month, d=day, v=value))
    
            def alter_move(num):
                for i in range(num):
                    produce_alter_move(node1, name)
    
            def alter_drop(num):
                for i in range(num):
                    partition = random.choice([201903, 201904])
                    drach = random.choice(["drop", "detach"])
                    node1.query("ALTER TABLE {} {} PARTITION {}".format(name, drach, partition))
    
            insert(100)
            p = Pool(15)
            tasks = []
            for i in range(5):
                tasks.append(p.apply_async(insert, (100,)))
                tasks.append(p.apply_async(alter_move, (100,)))
                tasks.append(p.apply_async(alter_drop, (100,)))
    
            for task in tasks:
>               task.get(timeout=60)

test_multiple_disks/test.py:929: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <multiprocessing.pool.ApplyResult object at 0x7f6250484b00>, timeout = 60

    def get(self, timeout=None):
        self.wait(timeout)
        if not self.ready():
>           raise TimeoutError
E           multiprocessing.context.TimeoutError

/usr/lib/python3.6/multiprocessing/pool.py:640: TimeoutError
_ test_concurrent_alter_move_and_drop[concurrently_dropping_replicated_mt-ReplicatedMergeTree('/clickhouse/concurrently_dropping_replicated_mt', '1')] _

start_cluster = <helpers.cluster.ClickHouseCluster object at 0x7f625cbbdba8>
name = 'concurrently_dropping_replicated_mt'
engine = "ReplicatedMergeTree('/clickhouse/concurrently_dropping_replicated_mt', '1')"

    @pytest.mark.parametrize("name,engine", [
        ("concurrently_dropping_mt", "MergeTree()"),
        ("concurrently_dropping_replicated_mt",
         "ReplicatedMergeTree('/clickhouse/concurrently_dropping_replicated_mt', '1')",),
    ])
    def test_concurrent_alter_move_and_drop(start_cluster, name, engine):
        try:
            node1.query("""
                CREATE TABLE {name} (
                    EventDate Date,
                    number UInt64
                ) ENGINE = {engine}
                ORDER BY tuple()
                PARTITION BY toYYYYMM(EventDate)
                SETTINGS storage_policy='jbods_with_external'
            """.format(name=name, engine=engine))
    
            values = list({random.randint(1, 1000000) for _ in range(0, 1000)})
    
            def insert(num):
                for i in range(num):
                    day = random.randint(11, 30)
                    value = values.pop()
                    month = '0' + str(random.choice([3, 4]))
                    node1.query("INSERT INTO {} VALUES(toDate('2019-{m}-{d}'), {v})".format(name, m=month, d=day, v=value))
    
            def alter_move(num):
                for i in range(num):
                    produce_alter_move(node1, name)
    
            def alter_drop(num):
                for i in range(num):
                    partition = random.choice([201903, 201904])
                    drach = random.choice(["drop", "detach"])
                    node1.query("ALTER TABLE {} {} PARTITION {}".format(name, drach, partition))
    
            insert(100)
            p = Pool(15)
            tasks = []
            for i in range(5):
                tasks.append(p.apply_async(insert, (100,)))
                tasks.append(p.apply_async(alter_move, (100,)))
                tasks.append(p.apply_async(alter_drop, (100,)))
    
            for task in tasks:
>               task.get(timeout=60)
```

**Comment by alexey-milovidov** [comment]
We have a dataset with all failed tests.
I've run a query:

```
SELECT test_name, count() AS c, max(check_start_time) AS last_time, argMin(commit_sha, check_start_time) AS first_seen, argMax(commit_sha, check_start_time) AS last_seen FROM `gh-data`.checks
WHERE
    test_status = 'FAIL'
    AND check_name LIKE 'Integration%'
    AND pull_request_number = 0
    AND check_start_time >= now() - INTERVAL 100 DAY
GROUP BY test_name
HAVING last_time >= now() - INTERVAL 10 DAY
ORDER BY last_time DESC
LIMIT 100
```

(list tests that have failed in the latest 10 days)
and get the following result:

| test_name | c | last_time | first_seen | last_seen | 
|:-:|-:|:-:|:-|:-|
| test_odbc_interaction/test.py::test_many_connections | 1 | 2021-04-13 17:13:43 | 6d8efe5d5b3efe303cc966657862722f4952dbc7 | 6d8efe5d5b3efe303cc966657862722f4952dbc7 |
| test_odbc_interaction/test.py::test_odbc_postgres_conversions | 1 | 2021-04-13 17:13:43 | 6d8efe5d5b3efe303cc966657862722f4952dbc7 | 6d8efe5d5b3efe303cc966657862722f4952dbc7 |
| test_odbc_interaction/test.py::test_concurrent_queries | 1 | 2021-04-13 17:13:43 | 6d8efe5d5b3efe303cc966657862722f4952dbc7 | 6d8efe5d5b3efe303cc966657862722f4952dbc7 |
| test_odbc_interaction/test.py::test_odbc_cyrillic_with_varchar | 1 | 2021-04-13 17:13:43 | 6d8efe5d5b3efe303cc966657862722f4952dbc7 | 6d8efe5d5b3efe303cc966657862722f4952dbc7 |
| test_host_ip_change/test.py::test_user_access_ip_change[node1] | 9 | 2021-04-12 21:37:25 | 7364fe4d4c2d9be3ad3590cb778198a98a16cf61 | 56f0e290de50ecb6cdf6a4055966c620f5f1c15e |
| test_host_ip_change/test.py::test_user_access_ip_change[node0] | 9 | 2021-04-12 21:37:25 | 7364fe4d4c2d9be3ad3590cb778198a98a16cf61 | 56f0e290de50ecb6cdf6a4055966c620f5f1c15e |
| test_merge_tree_s3_restore/test.py::test_restore_another_bucket_path | 66 | 2021-04-12 21:37:25 | f59dc7f40dc745def10ef26f78fc9d4d5fc60a08 | 56f0e290de50ecb6cdf6a4055966c620f5f1c15e |
| test_storage_kafka/test.py::test_kafka_virtual_columns2 | 2 | 2021-04-11 18:49:02 | 9f04683923502e4f287552ccbe2f81baad1d067d | aa6183af864af3b3b3a77ab250eb36a71041dd73 |
| test_materialize_mysql_database/test.py::test_insert_with_modify_binlog_checksum_5_7[clickhouse_node0] | 8 | 2021-04-11 12:28:47 | b2473686171e42efec83f7ecee7e463170f33eb0 | 0f6fc5bd7272a8eae045f22aeaefad29ed8ac8fd |
| test_materialize_mysql_database/test.py::test_materialize_database_ddl_with_mysql_5_7[clickhouse_node1] | 8 | 2021-04-11 12:28:47 | b2473686171e42efec83f7ecee7e463170f33eb0 | 0f6fc5bd7272a8eae045f22aeaefad29ed8ac8fd |
| test_materialize_mysql_database/test.py::test_materialize_database_ddl_with_empty_transaction_5_7[clickhouse_node0] | 8 | 2021-04-11 12:28:47 | b2473686171e42efec83f7ecee7e463170f33eb0 | 0f6fc5bd7272a8eae045f22aeaefad29ed8ac8fd |
| test_materialize_mysql_database/test.py::test_insert_with_modify_binlog_checksum_5_7[clickhouse_node1] | 8 | 2021-04-11 12:28:47 | b2473686171e42efec83f7ecee7e463170f33eb0 | 0f6fc5bd7272a8eae045f22aeaefad29ed8ac8fd |
| test_materialize_mysql_database/test.py::test_network_partition_5_7[clickhouse_node0] | 8 | 2021-04-11 12:28:47 | b2473686171e42efec83f7ecee7e463170f33eb0 | 0f6fc5bd7272a8eae045f22aeaefad29ed8ac8fd |
| test_materialize_mysql_database/test.py::test_materialize_database_err_sync_user_privs_5_7[clickhouse_node0] | 8 | 2021-04-11 12:28:47 | b2473686171e42efec83f7ecee7e463170f33eb0 | 0f6fc5bd7272a8eae045f22aeaefad29ed8ac8fd |
| test_materialize_mysql_database/test.py::test_materialize_database_ddl_with_mysql_5_7[clickhouse_node0] | 8 | 2021-04-11 12:28:47 | b2473686171e42efec83f7ecee7e463170f33eb0 | 0f6fc5bd7272a8eae045f22aeaefad29ed8ac8fd |
| test_materialize_mysql_database/test.py::test_select_without_columns_5_7[clickhouse_node0] | 8 | 2021-04-11 12:28:47 | b2473686171e42efec83f7ecee7e463170f33eb0 | 0f6fc5bd7272a8eae045f22aeaefad29ed8ac8fd |
| test_materialize_mysql_database/test.py::test_mysql_kill_sync_thread_restore_5_7[clickhouse_node0] | 8 | 2021-04-11 12:28:47 | b2473686171e42efec83f7ecee7e463170f33eb0 | 0f6fc5bd7272a8eae045f22aeaefad29ed8ac8fd |
| test_materialize_mysql_database/test.py::test_materialize_database_dml_with_mysql_5_7[clickhouse_node0] | 8 | 2021-04-11 12:28:47 | b2473686171e42efec83f7ecee7e463170f33eb0 | 0f6fc5bd7272a8eae045f22aeaefad29ed8ac8fd |
| test_materialize_mysql_database/test.py::test_mysql_kill_sync_thread_restore_5_7[clickhouse_node1] | 8 | 2021-04-11 12:28:47 | b2473686171e42efec83f7ecee7e463170f33eb0 | 0f6fc5bd7272a8eae045f22aeaefad29ed8ac8fd |
| test_materialize_mysql_database/test.py::test_network_partition_5_7[clickhouse_node1] | 8 | 2021-04-11 12:28:47 | b2473686171e42efec83f7ecee7e463170f33eb0 | 0f6fc5bd7272a8eae045f22aeaefad29ed8ac8fd |
| test_materialize_mysql_database/test.py::test_materialize_database_dml_with_mysql_5_7[clickhouse_node1] | 7 | 2021-04-11 12:28:47 | b2473686171e42efec83f7ecee7e463170f33eb0 | 0f6fc5bd7272a8eae045f22aeaefad29ed8ac8fd |
| test_materialize_mysql_database/test.py::test_select_without_columns_5_7[clickhouse_node1] | 9 | 2021-04-11 12:28:47 | 2c71b997de3f49de80802ced4dd19787f344486a | 0f6fc5bd7272a8eae045f22aeaefad29ed8ac8fd |
| test_materialize_mysql_database/test.py::test_materialize_database_err_sync_user_privs_5_7[clickhouse_node1] | 8 | 2021-04-11 12:28:47 | b2473686171e42efec83f7ecee7e463170f33eb0 | 0f6fc5bd7272a8eae045f22aeaefad29ed8ac8fd |
| test_materialize_mysql_database/test.py::test_materialize_database_ddl_with_empty_transaction_5_7[clickhouse_node1] | 8 | 2021-04-11 12:28:47 | b2473686171e42efec83f7ecee7e463170f33eb0 | 0f6fc5bd7272a8eae045f22aeaefad29ed8ac8fd |
| test_materialize_mysql_database/test.py::test_mysql_killed_while_insert_5_7[clickhouse_node1] | 8 | 2021-04-11 12:28:47 | b2473686171e42efec83f7ecee7e463170f33eb0 | 0f6fc5bd7272a8eae045f22aeaefad29ed8ac8fd |
| test_materialize_mysql_database/test.py::test_mysql_killed_while_insert_5_7[clickhouse_node0] | 8 | 2021-04-11 12:28:47 | b2473686171e42efec83f7ecee7e463170f33eb0 | 0f6fc5bd7272a8eae045f22aeaefad29ed8ac8fd |
| test_odbc_interaction/test.py::test_bridge_dies_with_parent | 149 | 2021-04-11 00:58:00 | 015e392bb7fa43989f4d58592759d94e9410e05d | 495c6e03aa9437dac3cd7a44ab3923390bef9982 |
| test_s3_zero_copy_replication/test.py::test_s3_zero_copy_on_hybrid_storage | 2 | 2021-04-08 22:05:19 | 51bbc9c9ba81ba490ff64810a271545d58d1e798 | 1c28878f5d2144c5fd59deeb0c1e41ec49068269 |
| test_s3_zero_copy_replication/test.py::test_s3_zero_copy_replication[s3] | 1 | 2021-04-08 22:05:19 | 1c28878f5d2144c5fd59deeb0c1e41ec49068269 | 1c28878f5d2144c5fd59deeb0c1e41ec49068269 |
| test_library_bridge/test.py::test_load_keys | 1 | 2021-04-06 12:15:33 | 4402ec31015cff7b7bb6e462de93ae667e624195 | 4402ec31015cff7b7bb6e462de93ae667e624195 |
| test_library_bridge/test.py::test_null_values | 1 | 2021-04-06 12:15:33 | 4402ec31015cff7b7bb6e462de93ae667e624195 | 4402ec31015cff7b7bb6e462de93ae667e624195 |
| test_library_bridge/test.py::test_load_all | 1 | 2021-04-06 12:15:33 | 4402ec31015cff7b7bb6e462de93ae667e624195 | 4402ec31015cff7b7bb6e462de93ae667e624195 |
| test_library_bridge/test.py::test_load_all_many_rows | 1 | 2021-04-06 12:15:33 | 4402ec31015cff7b7bb6e462de93ae667e624195 | 4402ec31015cff7b7bb6e462de93ae667e624195 |
| test_library_bridge/test.py::test_load_ids | 1 | 2021-04-06 12:15:33 | 4402ec31015cff7b7bb6e462de93ae667e624195 | 4402ec31015cff7b7bb6e462de93ae667e624195 |
| test_materialize_mysql_database/test.py::test_system_parts_table[clickhouse_node0] | 1 | 2021-04-04 17:37:19 | caaa19edb8d56c2e1621413a8e44a38719ec5bcd | caaa19edb8d56c2e1621413a8e44a38719ec5bcd |
| test_materialize_mysql_database/test.py::test_network_partition_8_0[clickhouse_node1] | 1 | 2021-04-04 17:37:19 | caaa19edb8d56c2e1621413a8e44a38719ec5bcd | caaa19edb8d56c2e1621413a8e44a38719ec5bcd |
| test_materialize_mysql_database/test.py::test_clickhouse_killed_while_insert_8_0[clickhouse_node0] | 1 | 2021-04-04 17:37:19 | caaa19edb8d56c2e1621413a8e44a38719ec5bcd | caaa19edb8d56c2e1621413a8e44a38719ec5bcd |
| test_materialize_mysql_database/test.py::test_network_partition_8_0[clickhouse_node0] | 1 | 2021-04-04 17:37:19 | caaa19edb8d56c2e1621413a8e44a38719ec5bcd | caaa19edb8d56c2e1621413a8e44a38719ec5bcd |
| test_materialize_mysql_database/test.py::test_clickhouse_killed_while_insert_8_0[clickhouse_node1] | 2 | 2021-04-04 17:37:19 | 4d0973cf250707a8e55899a1e77db6aa53dfe87d | caaa19edb8d56c2e1621413a8e44a38719ec5bcd |
| test_materialize_mysql_database/test.py::test_insert_with_modify_binlog_checksum_8_0[clickhouse_node0] | 1 | 2021-04-04 17:37:19 | caaa19edb8d56c2e1621413a8e44a38719ec5bcd | caaa19edb8d56c2e1621413a8e44a38719ec5bcd |
| test_materialize_mysql_database/test.py::test_mysql_killed_while_insert_8_0[clickhouse_node1] | 1 | 2021-04-04 17:37:19 | caaa19edb8d56c2e1621413a8e44a38719ec5bcd | caaa19edb8d56c2e1621413a8e44a38719ec5bcd |
| test_materialize_mysql_database/test.py::test_mysql_killed_while_insert_8_0[clickhouse_node0] | 1 | 2021-04-04 17:37:19 | caaa19edb8d56c2e1621413a8e44a38719ec5bcd | caaa19edb8d56c2e1621413a8e44a38719ec5bcd |
| test_materialize_mysql_database/test.py::test_insert_with_modify_binlog_checksum_8_0[clickhouse_node1] | 1 | 2021-04-04 17:37:19 | caaa19edb8d56c2e1621413a8e44a38719ec5bcd | caaa19edb8d56c2e1621413a8e44a38719ec5bcd |
| test_materialize_mysql_database/test.py::test_select_without_columns_8_0[clickhouse_node0] | 1 | 2021-04-04 17:37:19 | caaa19edb8d56c2e1621413a8e44a38719ec5bcd | caaa19edb8d56c2e1621413a8e44a38719ec5bcd |
| test_materialize_mysql_database/test.py::test_materialize_database_dml_with_mysql_8_0[clickhouse_node1] | 1 | 2021-04-04 17:37:19 | caaa19edb8d56c2e1621413a8e44a38719ec5bcd | caaa19edb8d56c2e1621413a8e44a38719ec5bcd |
| test_materialize_mysql_database/test.py::test_system_parts_table[clickhouse_node1] | 1 | 2021-04-04 17:37:19 | caaa19edb8d56c2e1621413a8e44a38719ec5bcd | caaa19edb8d56c2e1621413a8e44a38719ec5bcd |
| test_materialize_mysql_database/test.py::test_materialize_database_dml_with_mysql_8_0[clickhouse_node0] | 1 | 2021-04-04 17:37:19 | caaa19edb8d56c2e1621413a8e44a38719ec5bcd | caaa19edb8d56c2e1621413a8e44a38719ec5bcd |
| test_materialize_mysql_database/test.py::test_select_without_columns_8_0[clickhouse_node1] | 23 | 2021-04-04 17:37:19 | 176358f0b4bfcafa12887b2b6cd8e7fb397a56e3 | caaa19edb8d56c2e1621413a8e44a38719ec5bcd |
| test_materialize_mysql_database/test.py::test_system_tables_table[clickhouse_node1] | 1 | 2021-04-04 17:37:19 | caaa19edb8d56c2e1621413a8e44a38719ec5bcd | caaa19edb8d56c2e1621413a8e44a38719ec5bcd |
| test_materialize_mysql_database/test.py::test_materialize_database_ddl_with_empty_transaction_8_0[clickhouse_node0] | 1 | 2021-04-04 17:37:19 | caaa19edb8d56c2e1621413a8e44a38719ec5bcd | caaa19edb8d56c2e1621413a8e44a38719ec5bcd |
| test_materialize_mysql_database/test.py::test_materialize_database_ddl_with_mysql_8_0[clickhouse_node1] | 1 | 2021-04-04 17:37:19 | caaa19edb8d56c2e1621413a8e44a38719ec5bcd | caaa19edb8d56c2e1621413a8e44a38719ec5bcd |
| test_materialize_mysql_database/test.py::test_system_tables_table[clickhouse_node0] | 1 | 2021-04-04 17:37:19 | caaa19edb8d56c2e1621413a8e44a38719ec5bcd | caaa19edb8d56c2e1621413a8e44a38719ec5bcd |
| test_materialize_mysql_database/test.py::test_materialize_database_ddl_with_empty_transaction_8_0[clickhouse_node1] | 1 | 2021-04-04 17:37:19 | caaa19edb8d56c2e1621413a8e44a38719ec5bcd | caaa19edb8d56c2e1621413a8e44a38719ec5bcd |
| test_materialize_mysql_database/test.py::test_materialize_database_ddl_with_mysql_8_0[clickhouse_node0] | 1 | 2021-04-04 17:37:19 | caaa19edb8d56c2e1621413a8e44a38719ec5bcd | caaa19edb8d56c2e1621413a8e44a38719ec5bcd |
| test_materialize_mysql_database/test.py::test_materialize_database_err_sync_user_privs_8_0[clickhouse_node1] | 1 | 2021-04-04 17:37:19 | caaa19edb8d56c2e1621413a8e44a38719ec5bcd | caaa19edb8d56c2e1621413a8e44a38719ec5bcd |
| test_materialize_mysql_database/test.py::test_mysql_kill_sync_thread_restore_8_0[clickhouse_node1] | 1 | 2021-04-04 17:37:19 | caaa19edb8d56c2e1621413a8e44a38719ec5bcd | caaa19edb8d56c2e1621413a8e44a38719ec5bcd |
| test_materialize_mysql_database/test.py::test_materialize_database_err_sync_user_privs_8_0[clickhouse_node0] | 1 | 2021-04-04 17:37:19 | caaa19edb8d56c2e1621413a8e44a38719ec5bcd | caaa19edb8d56c2e1621413a8e44a38719ec5bcd |
| test_materialize_mysql_database/test.py::test_mysql_kill_sync_thread_restore_8_0[clickhouse_node0] | 1 | 2021-04-04 17:37:19 | caaa19edb8d56c2e1621413a8e44a38719ec5bcd | caaa19edb8d56c2e1621413a8e44a38719ec5bcd |

**Comment by alexey-milovidov** [comment]
From this result I see that the test `tests test_ttl_replicated` is being fixed.

**Comment by alexey-milovidov** [comment]
And the most flaky test is `test_materialize_mysql_database`, FYI @BohuTANG @zhang2014

**Comment by alexey-milovidov** [comment]
FYI `test_multiple_disks` is fixed in #16235.
